pkg/database/mysql/tables.go

```diff
@@ -30,10 +30,12 @@ func (m *MysqlConnection) ListTables() ([]string, error) {
 
 func (m *MysqlConnection) ListTableForeignKeys(databaseName string, tableName string) ([]*types.ForeignKey, error) {
 	query := `select
-	kcu.COLUMN_NAME, kcu.CONSTRAINT_NAME, kcu.REFERENCED_TABLE_NAME, kcu.REFERENCED_COLUMN_NAME
+	kcu.COLUMN_NAME, kcu.CONSTRAINT_NAME, kcu.REFERENCED_TABLE_NAME, kcu.REFERENCED_COLUMN_NAME, rc.DELETE_RULE
 	from information_schema.KEY_COLUMN_USAGE kcu
 	inner join information_schema.TABLE_CONSTRAINTS tc
-	on tc.CONSTRAINT_NAME = kcu.CONSTRAINT_NAME
+  	  on tc.CONSTRAINT_NAME = kcu.CONSTRAINT_NAME
+	inner join information_schema.REFERENTIAL_CONSTRAINTS rc
```

Comment:
this additional join is a 1-to-1 mapping and it always has a matching row?

Reply:
Appears to be valid so far in testing. This is internal-to-pg behavior though. I think we should wait for a test case that invalidates this theory before proposing a change.